app/controllers/experts_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ExpertsController < ApplicationController
+  before_action :find_expert
+
+  skip_before_action :authenticate_user!
+
+  layout 'experts'
+
+  def diagnosis
+    associations = [visit: [:visitee, :advisor, facility: [:company]], diagnosed_needs: [:selected_assistance_experts]]
+    @diagnosis = Diagnosis.available_for_expert(@expert).includes(associations).find(params[:diagnosis_id])
+    UseCases::UpdateExpertViewedPageAt.perform(diagnosis_id: params[:diagnosis_id].to_i, expert_id: @expert.id)
```

Comment:
is the to_i really necessary ?

Reply:
For testing purposes, but would work otherwise